src/app/public/modules/data-manager/data-manager-toolbar/data-manager-toolbar.component.spec.ts

```diff
@@ -91,6 +97,38 @@ describe('SkyDataManagerToolbarComponent', () => {
   let modalServiceInstance: MockModalService;
   let viewConfig: SkyDataViewConfig;
 
+  function setSearchInput(text: string) {
+    let inputEvent = document.createEvent('Event');
+    let params = {
+      bubbles: false,
+      cancelable: false
+    };
+    inputEvent.initEvent('input', params.bubbles, params.cancelable);
+
+    let changeEvent = document.createEvent('Event');
+    changeEvent.initEvent('change', params.bubbles, params.cancelable);
+    let inputEl = dataManagerToolbarFixture.debugElement.query(By.css('input'));
+    inputEl.nativeElement.value = text;
+
+    inputEl.nativeElement.dispatchEvent(inputEvent);
+    dataManagerToolbarFixture.detectChanges();
+
+    inputEl.nativeElement.dispatchEvent(changeEvent);
+    dataManagerToolbarFixture.detectChanges();
+  }
+
+  function triggerSearchInputEnter() {
+    let inputEl = dataManagerToolbarFixture.debugElement.query(By.css('.sky-search-container input'));
+    inputEl.triggerEventHandler('keyup', { which: 13});
+    dataManagerToolbarFixture.detectChanges();
+  }
+
+  function triggerSearchApplyButton() {
```

Comment:
Functions need a return type.

Reply:
Fixed all three of the helper functions here.